torch/_dynamo/test_case.py

```diff
@@ -178,13 +177,13 @@ def tearDownClass(cls) -> None:
     @classmethod
     def setUpClass(cls) -> None:
         # Skip test if python versions doesn't match
-        normalized_path = pathlib.PurePath("dynamo/cpython").as_posix()
-        regex = re.escape(normalized_path) + r"\b\d+_\d{2}\b"
+        prefix = os.path.join("dynamo", "cpython") + os.path.sep
+        regex = prefix + r"\d_\d{2}"
```

Comment:
We don't need to escape prefix here? Couldn't this regex fail on windows otherwise?

Reply:
Yeah, we do!